sdk/cosmos/azure-cosmos/test/test_query_async.py

```diff
@@ -936,6 +936,91 @@ async def query_items(database):
         self.client.client_connection.connection_policy.RetryOptions = old_retry
         await self.created_db.delete_container(created_collection.id)
 
+    async def test_query_request_params_none_retry_policy(self):
+        created_collection = await self.created_db.create_container_if_not_exists(
+            id="query_request_params_none_retry_policy_" + str(uuid.uuid4()),
+            partition_key=PartitionKey(path="/pk")
+        )
+        items = [
+            {'id': str(uuid.uuid4()), 'pk': 'test', 'val': 5},
+            {'id': str(uuid.uuid4()), 'pk': 'test', 'val': 5},
+            {'id': str(uuid.uuid4()), 'pk': 'test', 'val': 5}
+        ]
+
+        for item in items:
+            await created_collection.create_item(body=item)
+
+        self.OriginalExecuteFunction = _retry_utility_async.ExecuteFunctionAsync
+        # Test session retry will properly push the exception when retries run out
+        _retry_utility_async.ExecuteFunctionAsync = self._MockExecuteFunctionSessionRetry
+        try:
+            query = "SELECT * FROM c"
+            items = created_collection.query_items(
+                query=query,
+                enable_cross_partition_query=True
+            )
+            fetch_results = [item async for item in items]
+        except exceptions.CosmosHttpResponseError as e:
+            assert e.status_code == 404
+            assert e.sub_status == 1002
+        _retry_utility_async.ExecuteFunctionAsync = self.OriginalExecuteFunction
```

Comment:
Why was this assignment needed when you set `ExecuteFunctionAsync` in line 968 again?

Please remove this line if it is not necessary.